### ISSUE TITLE
Should compile this plugin using the good old Java11, rather than the latest Java17

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/CompareDirectoriesGradlePlugin#24]
In the NineBreakHome Page project, I got an error while I tried the 0.1.6 of the CompareDirectoriesGradlePlguin

```
Cause: com/kazurayam/dircomp/CompareDirectoriesPlugin has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0

```

**Comment by kazurayam** [comment]
https://stackoverflow.com/questions/10382929/how-to-fix-java-lang-unsupportedclassversionerror-unsupported-major-minor-versi

https://en.wikipedia.org/wiki/Java_class_file#General_layout

Java SE 21 = 65 (0x41 hex),
Java SE 20 = 64 (0x40 hex),
Java SE 19 = 63 (0x3F hex),
Java SE 18 = 62 (0x3E hex),
Java SE 17 = 61 (0x3D hex),
Java SE 16 = 60 (0x3C hex),
Java SE 15 = 59 (0x3B hex),
Java SE 14 = 58 (0x3A hex),
Java SE 13 = 57 (0x39 hex),
Java SE 12 = 56 (0x38 hex),
Java SE 11 = 55 (0x37 hex),
Java SE 10 = 54 (0x36 hex),[[4]](https://en.wikipedia.org/wiki/Java_class_file#cite_note-4)
Java SE 9 = 53 (0x35 hex),[[5]](https://en.wikipedia.org/wiki/Java_class_file#cite_note-5)
Java SE 8 = 52 (0x34 hex),
Java SE 7 = 51 (0x33 hex),
Java SE 6.0 = 50 (0x32 hex),
Java SE 5.0 = 49 (0x31 hex),
JDK 1.4 = 48 (0x30 hex),
JDK 1.3 = 47 (0x2F hex),
JDK 1.2 = 46 (0x2E hex),
JDK 1.1 = 45 (0x2D hex).

**Comment by kazurayam** [comment]
Should I bring my default JDK to 11 from 17?

**Comment by kazurayam** [comment]
I released v0.1.7 which was compiled using Java11
